src/gui/uithememanager.cpp

```diff
@@ -42,9 +43,11 @@
 
 namespace
 {
-    const QString ICONS_DIR = QStringLiteral(":icons/");
-    const QString THEME_ICONS_DIR = QStringLiteral(":uitheme/icons/");
-    const QString CONFIG_FILE_NAME = QStringLiteral(":uitheme/config.json");
+    const char DEFAULT_ICONS_DIR[] = ":icons/";
```

Comment:
Why ditch `QString`? Seems you still use it everywhere.

Reply:
there was clazy warning of using non-pod types as global objects.

Reply:
> there was clazy warning of using non-pod types as global objects.

Damn Clazy is infuriating with its moralizing.

Reply:
@jagannatharjun 
You should still use `QString` here.

Reply:
From the Clazy docs:
>CTORS from globals are run before main, on library load, slowing down startup. This is more a problem for libraries, since usually the app won't use every feature the library provides, so it's a waste of resources to initialize CTORs from unused features.
It's tolerated to have global statics in executables, however, clazy doesn't know if it's compiling an executable or a library, so it's your job to run this check only on libraries. It doesn't harm, though, to also remove global statics from executables, because they're usually evil.

So the only "valuable" CONS for using global QString constants in qBittorrent is "because they're usually evil".
Should we really consider another way?
Or is there a way to disable this particular Clazy warning project-wide?